bin/tidepool_docker

```
@@ -0,0 +1,91 @@
+#!/bin/sh
```

I usually add the `-eu` flags to the shell, so it'd be `#!/bin/sh -eu`. The `-e` will immediately exit/fail the script if any command within it fails (rather that silently continuing on even after the error). The `-u` will fail on undefined environment variables (rather than simply replace with an empty string). I find both of these very useful to prevent unintended consequences.

---

Thanks for the tips.  I've never used those flags before.  I'll add them